data-service/src/integration/v1_graphql_entity.test.ts

```diff
@@ -7,7 +7,8 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const crossFetch = require('cross-fetch');
 
-const url = 'http://localhost:4000/v1/graphql';
+// const url = 'http://localhost:4000/v1/graphql';
+const url = 'http://localhost:4007/v1/graphql';
```

Comment:
![image](https://github.com/msupply-foundation/unified-codes/assets/8287373/a7d7db21-327d-45a0-8a4d-2cc5c0c35c18)

Any idea how to resolve this error?

Reply:
the `crossFetch`? can't you just use a different name for the variable?
`const cf = require('cross-fetch');`

though it would also be preferable to use `import` rather than `require`

`import cf from 'cross-fetch';`

Reply:
I'll put the `export {}` thing in for now :)